src/installation/guides/chroot.md

```diff
@@ -0,0 +1,232 @@
+# Installation via chroot (x86 or x86_64)
+
+## Prepare Filesystems
+
+These instructions can be followed on any running Linux system that has xbps
+installed, including an already working installation of Void or an official
+[live image](../live-images/prep.md). It is assumed that the host computer is
+connected to the Internet.
+
+[Partition your disks](../live-images/partitions.md) and format them using
+[mke2fs(8)](https://man.voidlinux.org/mke2fs.8),
+[mkfs.xfs(8)](https://man.voidlinux.org/mkfs.xfs.8),
+[mkfs.btrfs(8)](https://man.voidlinux.org/mkfs.btrfs.8) or whatever tools are
+necessary for your filesystem(s) of choice.
+
+[cfdisk(8)](https://man.voidlinux.org/cfdisk.8) and
+[fdisk(8)](https://man.voidlinux.org/fdisk.8) are available on live
+distributions for partitioning, but you may wish to use
+[gdisk(8)](https://man.voidlinux.org/gdisk.8) (from the package `gptfdisk`) or
+[parted(8)](https://man.voidlinux.org/parted.8) instead.
+
+> Note: For a UEFI booting system, make sure to create a UEFI System Partition
+> (ESP). The ESP should have the partition type "EFI System" (code `ef00`) and
+> be formatted as fat32 using
+> [mkfs.vfat(8)](https://man.voidlinux.org/mkfs.vfat.8).
+
+If unsure what partitions to create, create (1) 1GB in size with type `EFI
+System` (code EF00) (2) remainder of the drive with type `Linux Filesystem`
+(code 8300).
+
+Then format these partitions with fat32 and ext4 respectively.
+
+```
+# mkfs.vfat /dev/sda1
+# mkfs.ext4 /dev/sda2
+```
+
+### Create a New Root and Mount Filesystems
+
+This guide will assume the new root filesystem is mounted on `/mnt`. You may
+wish to mount it elsewhere.
+
+If using UEFI, mount the EFI System Partition as `/mnt/boot/efi`.
+
+For example, if `/dev/sda2` is to be mounted as `/` and `/dev/sda3` is to be
+booted as `/home` and `dev/sda1` is the EFI System Partition
```

Comment:
I think you forgot to remove the reference to the home partition here.

Reply:
yes I did